RoboJackets/urc-software
# Added websocket launch to bringup script

# Description
This PR does the following:
- ROS websocket gets launched in bringup.launch.py, integrates with web interface.
- Deleted old websocket launch script (behavior is in bringup.launch)

# Testing Steps (if relevant)
1. Run `$ ros2 launch urc_bringup websocket.launch.py`
2. Run the web interface (follow instructions in `urc_dashboard`)
3. Enter the correct host IP
4. Rosbridge status indicator should be green.

# Self Checklist
